Cargo.toml

```diff
@@ -40,6 +40,11 @@ serde_derive = "^1.0.80"
 shell2batch = "^0.2.4"
 toml = "^0.4.8"
 
+[[bin]]
+bench = false
+path = "src/main.rs"
+name = "cargo-bin"
```

Comment:
i would call it makers (make rust) since I think that point here is to make it a non cargo plugin so better remove cargo from its name. what do you think?

Reply:
Yeah, was chatting about that in the office with Rishab today. How about `runrs`? 📦

Reply:
ya something like that.
There is already a rust project with that name: https://github.com/Xion/runrs but its not published to crates.io so I think it should be ok...